test/helpers/browser.js

```diff
@@ -7,6 +7,7 @@ var vm = require('vm');
 
 function createContext() {
   var context = {
+    self: this,
```

Comment:
i wasnt able to figure out how to get the sourceMapped stacktrace, but `debug` is on for browserify, ill give it another try later

self is from the module [`formData`](https://github.com/form-data/form-data/commit/2c0a54532c22da345e85afd559a9c6a14ec8155b)
```js
.call(this,require("FT5ORs"),require("buffer").Buffer)
},{"./lib/auth":286,"./lib/cookies":287,"./lib/getProxyFromURI":288,"./lib/har":289,"./lib/helpers":290,"./lib/multipart":291,"./lib/oauth":292,"./lib/querystring":293,"./lib/redirect":294,"./lib/tunnel":295,"FT5ORs":268,"aws-sign2":35,"aws4":36,"buffer":60,"caseless":62,"extend":86,"forever-agent":88,"form-data":296,"hawk":110,"http":111,"http-signature":114,"https":119,"is-typedarray":127,"isstream":131,"mime-types":241,"stream":332,"stringstream":346,"url":390,"util":394,"zlib":49}],298:[function(require,module,exports){
(function (Buffer){
```

```js
296:[function(require,module,exports){
/* eslint-env browser */
module.exports = self.FormData;

},{}]
```